src/main/plugins/org.dita.xhtml/xsl/xslhtml/dita2htmlImpl.xsl

```diff
@@ -1151,16 +1151,10 @@ See the accompanying LICENSE file for applicable license.
 
 <!-- quotes - only do 1 level, no flip-flopping -->
 <xsl:template match="*[contains(@class, ' topic/q ')]" name="topic.q">
-  <span class="q">
+  <q>
     <xsl:call-template name="commonattributes"/>
-    <xsl:call-template name="getVariable">
```

Comment:
See e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q

> **Usage note**: Most modern browsers will automatically add quotation marks around text inside a `<q>` element. A style rule may be needed to add quotation marks in older browsers.

Thus I would prefer us **not** generate quotation marks into HTML output and handle the generation in CSS. If users want hard coded quotation marks, they can override this XSLT.

Reply:
I'm confused ... this pull request gets rid of the variable use / gets rid of the hard coded quotation marks, and instead uses the native HTML `<q>` element. The browser's default CSS is now used to generate the quotation marks, and custom CSS can be used to change / remove the characters (or they can use XSLT as you suggested).

It looks like this is a comment disliking the code we should remove, rather than a comment on the new code?

Reply:
Sorry, my fried brain mixed red and green. Yes, this is the correct approach.